### ISSUE TITLE
lines on edges 

### ISSUE THREAD

**Issue by a1ex3d** [ywwynm/Unity-EAC-shader#1]
Hello. I have some problems. When I use this shader on sphere, video play fine, but I see some edges like a cube edges. Then I "play" with scale variable (0.99) on u and v and some edges disappear (I dont see them) but not all.

**Comment by ywwynm** [comment]
@a1ex3d Yes, I found them too. This should be a precision issue. I've tried some ways to solve it but none works. Can you give any suggestions?

**Comment by Ming0608** [comment]
Hello I have some problems. I use this shader on PC,Video play fine.but not on the Android side
